_episodes/40.md

```diff
@@ -0,0 +1,443 @@
+---
+layout: episode
+title:  "40: Trino's cold as ice!"
+date: 2022-09-07
+tags: iceberg apache datalake table format sql schema evolution partition optimize
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the week"
+     desc: "Latest features in Apache Iceberg and the Iceberg connector"
+     time: 0
+   - title: "PR of the week"
+     desc: ""
+     time: 0
+   - title: "Demo"
+     desc: "Merge tables with Iceberg and reading the data in Trino"
+     time: 0
+   - title: "Question of the week"
+     desc: ""
+     time: 0
+---
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/15/trino-iceberg.png"/><br/>
+Looks like Commander Bun Bun is safe on this Iceberg<br/>
+<a href="https://joshdata.me/iceberger.html">https://joshdata.me/iceberger.html</a>
+</p>
+
```

Comment:
Should we have a hosts section going forward?

Reply:
I suppose it makes sense now that we switch out.

Reply:
done